app/src/main/java/com/example/triptracker/view/home/HomeUtils.kt

```diff
@@ -69,7 +74,10 @@ fun DisplayItinerary(
 
   var readyToDisplay by remember { mutableStateOf(false) }
   var profile by remember { mutableStateOf(UserProfile("")) }
-
+  if (test) {
+    readyToDisplay = true
+    profile = UserProfile("test@gmail.com", "Test User", "test", "test bio")
+  }
```

Comment:
not the best way to do it. Maybe add a constant in the view model ?

Reply:
You're right. I'll put it in a dummyUserProfile above